charts/matrix-stack/configs/matrix-rtc/sfu/config-overrides.yaml.tpl

```diff
@@ -55,4 +55,9 @@ key_file: /secrets/{{ (printf "/secrets/%s"
 key_file: /conf/keys.yaml
 {{- end }}
 
+{{- if $root.Values.matrixRTC.restrictRoomCreationToLocalUsers }}
```

Comment:
I don't think this needs to be gated. With `LIVEKIT_FULL_ACCESS_HOMESERVERS=*` the authoriser will create rooms for any homeserver and this can still be set